src/context/Auth.tsx

```diff
@@ -14,25 +14,16 @@ export interface AuthContextType {
 export async function bootstrapUser() {
     let user = null;
 
-    const token = await auth.getToken();
-    if (token) {
-        const accountID = await auth.getAccountID();
-        if (accountID) {
-            // TODO - This is basically what is done over in src/services/auth.ts signin function
-            //    we should clean it up so this is shared
-            setAuthHeader(token, accountID);
-            await auth
-                .getAccountDetails(`accounts/${accountID}`)
-                .then((accountDetails) => {
-                    user = accountDetails.display_name;
-                })
-                .catch(() => {
-                    user = null;
-                });
+    const authDetails = auth.getAuthDetails();
 
-            return user;
+    if (authDetails?.session) {
+        if (isFuture(new Date(authDetails.session.expires_at))) {
+            user = authDetails.user?.display_name;
+        } else {
+            user = null;
```

Comment:
We now have all the details in the local storage so we no longer have to call the server.